packages/core/examples/cardExample.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+import { Card, Classes, Elevetaion, Slider, Switch } from "@blueprintjs/core";
+import { BaseExample } from "@blueprintjs/docs";
+
+export interface ICardExampleState {
+    elevation?: Elevetaion;
```

Comment:
spelling: `Elevation`

this doesn't actually compile, but webpack errors don't fail the build 😢 
```
ERROR in ./packages/core/examples/cardExample.tsx
(10,25): error TS2305: Module '"/home/ubuntu/blueprint/packages/site-docs/node_modules/@blueprintjs/core/dist/index"' has no exported member 'Elevetaion'.
```